docs/api/javascript.md

```diff
@@ -41,7 +41,7 @@ See [Pagy Navs](javascript/navs.md)
 
 +++ `pagy*_combo_nav_js`
 
-![bootstrap_combo_nav_js](/docs/assets/images/bootstrap_combo_nav_js-g.png)
+![bootstrap_combo_nav_js](/docs/assets/images/bootstrap_combo_nav_js.png)
 
 * Navigation and pagination info combined.
 
```

Comment:
This file uses also the nav_js image, so we should change the image to the white version, adding also the same alt description.

Reply:
fixed with [b8bf632](https://github.com/ddnexus/pagy/pull/453/commits/b8bf6328d00d2d23cf028455247035edddf7139c)